triemux/mux.go

```diff
@@ -37,6 +40,13 @@ func NewMux() *Mux {
 // If the routing table is empty, return a 503.
 func (mux *Mux) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	if mux.count == 0 {
+		defer func() {
```

Comment:
Do you actually need the `defer func()` here? Could you just inline this code after `w.WriteHeader(http.StatusServiceUnavailable)`? Or is there a chance that will panic?

Reply:
True, we don't need to `defer` this.